src/client/pages/Welcome.tsx

```diff
@@ -44,11 +48,11 @@ export const Welcome = ({
       current={CONSENTS_PAGES.PASSWORD}
       showContinueButton={false}
     >
-      <p css={[text, greyBorderTop, autoRow()]}>
+      <p css={[text, greyBorderTop, passwordInstructions, autoRow()]}>
         {passwordSet
           ? 'Password already set for '
-          : 'Please create a password for '}{' '}
-        {<span css={emailSpan}>{email}</span> || 'your new account'}
+          : 'Please create a password for '}
+        {email ? <span css={emailSpan}>{email}</span> : 'your new account'}
```

Comment:
The "your new account" text would never show! 😬

Reply:
good spot, maybe worth extending/adding an assertion in one of our tests to cover it along with this fix

there's already an assertion checking the `Password already set for` message here: https://github.com/guardian/gateway/blob/main/cypress%2Fintegration%2Fmocked%2Fwelcome.spec.ts#L135